blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -797,13 +799,13 @@ public void parameterizedFreestyleTestWithoutDefaultParam() throws Exception {
                 ), 400);
     }
 
+    @Ignore
```

Comment:
Yep, I think i might have been the one that added that test. Its a classpath issue that is causing it to fnial, havent figured out why.